code/game/objects/structures/customplaque.dm

```diff
@@ -0,0 +1,146 @@
+/obj/structure/customplaque //This is a plaque you can craft with gold, then permanently engrave a title and description on, with a fountain pen.
+	icon = 'icons/obj/decals.dmi'
+	icon_state = "blankplaque"
+	name = "blank plaque"
+	desc = "A blank plaque, use a fancy pen to engrave it. It can be detatched from the wall with a wrench."
+	anchored = TRUE
+	opacity = 0
+	density = FALSE
+	layer = SIGN_LAYER
+	custom_materials = list(/datum/material/gold = 2000)
+	max_integrity = 200 //Twice as durable as regular signs.
+	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
+	var/unengraved = TRUE //Custom plaque structures and items both start "unengraved", once engraved with a fountain pen their text can't be altered again.
+
+/obj/item/customplaque
+	icon = 'icons/obj/decals.dmi'
+	icon_state = "blankplaque"
+	name = "blank plaque"
+	desc = "A blank plaque, use a fancy pen to engrave it. It can be placed on a wall."
+	w_class = WEIGHT_CLASS_NORMAL
+	custom_materials = list(/datum/material/gold = 2000)
+	max_integrity = 200
+	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+	var/plaque_path = /obj/structure/customplaque //This points the item to make the proper structure when placed on a wall.
+	var/unengraved = TRUE
+
+/obj/structure/customplaque/play_attack_sound(damage_amount, damage_type = BRUTE, damage_flag = NONE)
+	switch(damage_type)
+		if(BRUTE)
+			if(damage_amount)
+				playsound(loc, 'sound/weapons/smash.ogg', 80, TRUE)
+			else
+				playsound(loc, 'sound/weapons/tap.ogg', 50, TRUE)
+		if(BURN)
+			playsound(loc, 'sound/items/welder.ogg', 80, TRUE)
+
+/obj/structure/customplaque/attack_hand(mob/user)
+	. = ..()
+	if(.)
+		return
+	user.examinate(src)
+
+/obj/structure/customplaque/wrench_act(mob/living/user, obj/item/wrench/I)
+	. = ..()
+	user.visible_message("<span class='notice'>[user] starts removing [src]...</span>", \
+						 "<span class='notice'>You start unfastening [src].</span>")
+	I.play_tool_sound(src)
+	if(I.use_tool(src, user, 40))
+		playsound(src, 'sound/items/deconstruct.ogg', 50, TRUE)
+		user.visible_message("<span class='notice'>[user] unfastens [src].</span>", \
+							 "<span class='notice'>You unfasten [src].</span>")
+		var/obj/item/customplaque/CP = new (get_turf(user))
+		CP.name = name //Copy over the plaque structure variables to the plaque item we're creating when we unwrench it.
+		CP.desc = desc
+		CP.unengraved = unengraved
+		CP.obj_integrity = obj_integrity
+		CP.setDir(dir)
+		qdel(src) //The plaque structure on the wall goes poof and only the plaque item from unwrenching remains.
+	return TRUE
+
+/obj/structure/customplaque/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/pen/fountain) && unengraved)
```

Comment:
check ungraved first, istype is more complex, easiest checks always go first to save time

Reply:
Done.